ui/job-view/headerbars/SecondaryNavBar.jsx

```diff
@@ -284,6 +283,11 @@ class SecondaryNavBar extends React.PureComponent {
               } ${!duplicateJobsVisible ? 'strikethrough' : ''}`}
               tabIndex="0"
               role="button"
+              aria-label={
+                duplicateJobsVisible
+                  ? 'Hide duplicate jobs'
+                  : 'Show duplicate jobs'
+              }
```

Comment:
This is a good choice for the aria-label but the same logic is being duplicated for title.